resolwe/flow/managers/listener.py

```diff
@@ -420,6 +420,33 @@ def handle_finish(self, obj):
             },
         })
 
+    def handle_abort(self, obj):
+        """Handle an incoming ``Data`` abort processing request.
+
+        .. IMPORTANT::
+
+            This only makes manager's state consistent and doesn't
```

Comment:
It makes it consistent only in case the protocol was followed. In case the executor was executed twice (and the manager did not actually dispatch twice), this will actually make the state inconsistent.